src/wrapper/descriptors/class_desc.rs

```diff
@@ -23,6 +23,7 @@ impl<'a, 'b> Desc<'a, JClass<'a>> for JObject<'b> {
     }
 }
 
+/// This conversion assumes that the `GlobalRef` is a pointer to a class object.
```

Comment:
I see logic here, but there is a raw pointer (in terms of Rust) inside the `GlobalRef`. :thinking: 

@jechas01 What do you think?